src/libstd/sync/mpsc/oneshot.rs

```diff
@@ -136,7 +137,7 @@ impl<T> Packet<T> {
         }
     }
 
-    pub fn recv(&mut self) -> Result<T, Failure<T>> {
+    pub fn recv(&mut self, deadline: Option<Instant>) -> Result<T, Failure<T>> {
```

Comment:
Ah yeah these are just internal impl details that aren't exported, so it's fine to have these `pub`